### ISSUE TITLE
HoverCard: HighContrast Support

### ISSUE THREAD

**Issue by Boris-Em** [OfficeDev/office-ui-fabric-react#2857]
### Bug Report

- __Package version(s)__: 5.0
- __Browser and OS versions__: Edge

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes

Requested priority: Low

#### Expected behavior:

The HoverCard component should have a border around itself in high contrast dark mode.

#### Actual behavior:

![hovercard 1](https://user-images.githubusercontent.com/4613643/30461690-23dbe9b0-9975-11e7-8e24-12fa24ef0a3e.png)

**Comment by kristoferbrown** [comment]
Fixed by #2589
